### PR TITLE
Add periodic jobs to auto accept the bot's repo invitations

### PR DESCRIPTION
**Pull request by droslean**
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1763,3 +1763,69 @@ periodics:
...
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 12h
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```
Comment (alvaroaleman) [inline ci-operator/jobs/infra-periodics.yaml:1775]: Can we add the infra labels so we get alerted when this fails?

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:1775]: Yea, that was my intention too, but I removed it for now to rehearse the job with dry-run.

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -1763,3 +1763,73 @@ periodics:
...
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
```
Comment (alvaroaleman) [inline ci-operator/jobs/infra-periodics.yaml:1775]: Shouldn't it also be rehearsable?

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:1775]: The rehearse will actually accept the invitations. For now, the rehearse tool is not intelligent enough to rehearse a job with extra/changed parameters (in this case `dry-run=true`).

File: ci-operator/jobs/infra-periodics.yaml [hunk 3]
```diff
@@ -1763,3 +1763,73 @@ periodics:
         - key: sa.config-updater.arm01.config
           path: kubeconfig
         secretName: build-farm-credentials
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
```
Comment (stevekuznetsov) [inline ci-operator/jobs/infra-periodics.yaml:1769]: Why do we need to clone `release`?

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:1769]: Copy-paste .... :P .Nice catch

File: ci-operator/jobs/infra-periodics.yaml (hunk 2 above)
Comment (alvaroaleman) [reply ci-operator/jobs/infra-periodics.yaml:1775]: Yeah, what's the problem with taht?

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:1771]: updated

**Comment by droslean** [comment]
```console
time="2021-05-06T14:53:15Z" level=info msg="ListCurrentUserRepoInvitations()" client=github
time="2021-05-06T14:53:15Z" level=info msg="Accepting invitation." bot-user=openshift-merge-robot invitation-id=14958392 repo=redhat-developer/devconsole-operator
time="2021-05-06T14:53:15Z" level=info msg="AcceptUserRepoInvitation(14958392)" client=github
time="2021-05-06T14:53:16Z" level=info msg="Accepting invitation." bot-user=openshift-merge-robot invitation-id=15290371 repo=redhat-developer/devconsole-api
time="2021-05-06T14:53:16Z" level=info msg="AcceptUserRepoInvitation(15290371)" client=github
time="2021-05-06T14:53:16Z" level=info msg="Accepting invitation." bot-user=openshift-merge-robot invitation-id=15290386 repo=redhat-developer/devconsole-git
time="2021-05-06T14:53:16Z" level=info msg="AcceptUserRepoInvitation(15290386)" client=github
time="2021-05-06T14:53:16Z" level=info msg="Accepting invitation." bot-user=openshift-merge-robot invitation-id=16502345 repo=gluster/gluster-mixins
time="2021-05-06T14:53:16Z" level=info msg="AcceptUserRepoInvitation(16502345)" client=github
time="2021-05-06T14:53:16Z" level=info msg="Accepting invitation." bot-user=openshift-merge-robot invitation-id=29218822 repo=opendatahub-io/opendatahub-operator
time="2021-05-06T14:53:16Z" level=info msg="AcceptUserRepoInvitation(29218822)" client=github 
```
:tada:

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**